### PR TITLE
docs: Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell`

### CODE REVIEW

**Comment by AnonymouX47** [comment]
Thanks so much, I'll review this soon.

**Review by AnonymouX47 — CHANGES_REQUESTED** [review]
Sorry it took me so long :( , I've been busy with #97.

Gone through all the changes. There's only one I found to be incorrect. Also, there are now conflicts.

Let me know if you're willing to make the necessary changes for a merge or not. If not, it's all okay :) , I'll handle them and merge locally.

Thanks.

**Comment by kianmeng** [comment]
@AnonymouX47 Done.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ For a more detailed explanation with examples see the guide at https://cbea.ms/g
 - Try to keep things (definitions, names, dictionary keys, list items, etc)
 
   - **grouped** (preferably by the most dominant criterium e.g object/definition type)
-  - **sorted** (preferrably lexicographically, with the exception of dependant definitions e.g decorators, metaclasses, baseclasses, etc)
+  - **sorted** (preferably lexicographically, with the exception of dependent definitions e.g decorators, metaclasses, baseclasses, etc)
```
Comment (AnonymouX47) [inline CONTRIBUTING.md:189]: Note to self: dependent -> depended-upon

File: docs/source/api/image.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ Enumerations
 Image Classes
 -------------
 
-Class Hierachy
+Class Hierarchy
 ^^^^^^^^^^^^^^
```
Comment (AnonymouX47) [inline docs/source/api/image.rst:53]: I missed this earlier, reST specifies that the underline (^^^...) be at least, as long as the section title.

Due to this, the docs are failing to build.

File: src/term_image/renderable/_renderable.py [hunk 3]
```diff
@@ -1017,7 +1017,7 @@ def _render_(self, render_data: RenderData, render_args: RenderArgs) -> Frame:
 
               * contain as many lines as :py:attr:`render_size.height
                 <term_image.geometry.Size.height>` i.e exactly
-                ``render_size.height - 1`` occurences of ``\\n`` (the newline sequence).
+                ``render_size.height - 1`` occurrences of ``\\n`` (the newline sequence).
```
Comment (AnonymouX47) [inline src/term_image/renderable/_renderable.py:1020]: This makes the line length 89 (> 88) and causes the lint check to fail. The line will require rewrapping.

File: tests/renderable/test_types.py [hunk 4]
```diff
@@ -100,7 +100,7 @@ def test_equality(self, args):
             (args, (99, 1000, Size(100, 100), " " * 100)),
         ],
     )
-    def test_unequality(self, args1, args2):
+    def test_inequality(self, args1, args2):
```
Comment (AnonymouX47) [inline tests/renderable/test_types.py:103]: "unequality" here is deliberate.

File: tests/test_padding.py [hunk 5]
```diff
@@ -334,7 +334,7 @@ def test_equality(self, args1, kwargs1, args2, kwargs2):
             ((1, 1), dict(fill="#"), (1, 1), dict(fill="*")),
         ],
     )
-    def test_unequality(self, args1, kwargs1, args2, kwargs2):
+    def test_inequality(self, args1, kwargs1, args2, kwargs2):
```
Comment (AnonymouX47) [inline tests/test_padding.py:337]: "unequality" here is deliberate.

File: tests/test_padding.py [hunk 6]
```diff
@@ -411,5 +411,5 @@ def test_equality(self, args1, kwargs1, args2, kwargs2):
             ((), dict(fill="#"), (), dict(fill="*")),
         ],
     )
-    def test_unequality(self, args1, kwargs1, args2, kwargs2):
+    def test_inequality(self, args1, kwargs1, args2, kwargs2):
```
Comment (AnonymouX47) [inline tests/test_padding.py:414]: "unequality" here is deliberate.

**Review by AnonymouX47 — CHANGES_REQUESTED** [review]
Thanks for adding in the new files.

I think most, if not all, of the changes I requested here will require manual correction.

Again, I'm totally fine with handling them. So, let me know if you'd prefer that.

If you would be handling them, I'll recommend you don't force-push this time :smiley:. I understand it was the simplest way out earlier due to the conflicts but it isn't always the best and I don't think it's necessary for these set of changes.

Thanks once again :smiley:.

**Review by AnonymouX47 — APPROVED** [review]
Thanks so much :heart:

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**